application-blog-ui/src/main/resources/Blog/CategorySheet.xml

```diff
@@ -47,7 +47,7 @@
 #set ($obj = $doc.getObject($blogCategoryClassname))
 #if ($obj)
   #getEntriesForCategory($doc.fullName $discard $totalEntries)
-  = Category: $services.rendering.escape($obj.getValue('name'), $doc.syntax) ($totalEntries posts) [[#toolImage('feed')&gt;&gt;Blog.CategoryRss||queryString="xpage=plain&amp;category=$escapetool.url($doc.fullName)" title="RSS"]] =
+  = $services.localization.render('xe.blog.sheet.category') $services.rendering.escape($obj.getValue('name'), $doc.syntax) ($totalEntries posts) [[#toolImage('feed')&gt;&gt;Blog.CategoryRss||queryString="xpage=plain&amp;category=$escapetool.url($doc.fullName)" title="RSS"]] =
```

Comment:
The 'xe' prefix is no longer valid. Make sure the translation exist in https://github.com/xwiki-contrib/application-blog/blob/master/application-blog-ui/src/main/resources/Blog/Translations.xml or create one.